src/server/JasmineGraphServer.cpp

```diff
@@ -424,7 +459,7 @@ void JasmineGraphServer::startRemoteWorkers(std::vector<int> workerPortsVector,
                         std::to_string(workerDataPortsVector.at(i)) + " --ENABLE_NMON " + enableNmon;
                 }
             }
-            server_logger.log(serverStartScript, "info");
+            server_logger.info(serverStartScript);
```

Comment:
Lets delete this log line. I do not see any point of having it here.

Reply:
Removed in next PR

Reply:
Ok